net/bitsrunlogin-go/files/bitsrunlogin-go.init

```diff
@@ -17,16 +17,20 @@ start_service() {
 
 	local enabled
 	config_get_bool enabled "config" "enabled" "0"
-	[ "$enabled" -eq "1" ] || exit 1
+	
+	if [ ! "$enabled" -eq "1" ]; then 
+	    return 0
+	fi
 
-	local domain usertype username password
+	local domain usertype username password acid
 	local enable_https skip_cert_verify timeout duration
 	local interfaces debug
 
 	config_get domain "config" "domain"
 	config_get usertype "config" "usertype"
 	config_get username "config" "username"
 	config_get password "config" "password"
+	config_get acid "config" "acid"
```

Comment:
is `acid` forced? I would suggest adding a default value for users who come from old configuration.